doc/stdlib/io-network.rst

```diff
@@ -900,53 +873,53 @@ Memory-mapped I/O
 Network I/O
 -----------
 
-.. function:: connect([host],port) -> TCPSocket
+.. function:: connect([host], port::Integer) -> TCPSocket
 
    .. Docstring generated from Julia source
 
    Connect to the host ``host`` on port ``port``\ .
 
-.. function:: connect(path) -> PipeEndpoint
+.. function:: connect(path::AbstractString) -> PipeEndpoint
 
    .. Docstring generated from Julia source
 
    Connect to the named pipe / UNIX domain socket at ``path``\ .
 
-.. function:: listen([addr,]port) -> TCPServer
+.. function:: listen([addr, ]port::Integer; backlog::Integer=BACKLOG_DEFAULT) -> TCPServer
```

Comment:
Looks like the last time this function was modified was in 2013, by @Keno in a96e16e5c386918dc701a333e5abfeb45501fdc6. Want to be a hero and doc what this does?